sql/core/src/test/resources/sql-tests/inputs/pgSQL/timestamp.sql

```diff
@@ -0,0 +1,247 @@
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 10 5:32PM 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('1997/02/10 17:32:01-0800');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('1997-02-10 17:32:01 PST');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb-10-1997 17:32:01 PST');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('02-10-1997 17:32:01 PST');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('19970210 173201 PST');
+-- set datestyle to ymd;
+-- INSERT INTO TIMESTAMP_TBL VALUES ('97FEB10 5:32:01PM UTC');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('97/02/10 17:32:01 UTC');
+-- reset datestyle;
+-- INSERT INTO TIMESTAMP_TBL VALUES ('1997.041 17:32:01 UTC');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('19970210 173201 America/New_York');
+-- this fails (even though TZ is a no-op, we still look it up)
+-- INSERT INTO TIMESTAMP_TBL VALUES ('19970710 173201 America/Does_not_exist');
+
+-- Check date conversion and date arithmetic
+-- INSERT INTO TIMESTAMP_TBL VALUES ('1997-06-10 18:32:01 PDT');
+
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 10 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 11 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 12 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 13 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 14 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 15 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 1997');
+
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 0097 BC');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 0097');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 0597');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 1097');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 1697');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 1797');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 1897');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 16 17:32:01 2097');
+
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 28 17:32:01 1996');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 29 17:32:01 1996');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Mar 01 17:32:01 1996');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Dec 30 17:32:01 1996');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Dec 31 17:32:01 1996');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Jan 01 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 28 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Feb 29 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Mar 01 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Dec 30 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Dec 31 17:32:01 1997');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Dec 31 17:32:01 1999');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Jan 01 17:32:01 2000');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Dec 31 17:32:01 2000');
+-- INSERT INTO TIMESTAMP_TBL VALUES ('Jan 01 17:32:01 2001');
+
+select * from TIMESTAMP_TBL;
```

Comment:
Ya. I know this is to check the insertion. But, in this case, we don't need this extra one because we have line 152.